voice/methods/conferences/getConferences.md

```diff
@@ -0,0 +1,123 @@
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter      | Description                                                                                                                      | Mandatory |
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| pageSize       | Specifies the max number of conferences that will be returned. Range: integer numbers from 1 to 1000. Default value is 1000.     | No        |
+| name           | Filter results by the `name` field.                                                                                              | No        |
+| minCreatedTime | Filter results to conferences which have a `createdTime` after or including `minCreatedTime` (in ISO8601 format).                | No        |
+| maxCreatedTime | Filter results to conferences which have a `createdTime` before or including `maxCreatedTime` (in ISO8601 format).               | No        |
+
+### Response Attributes
+
+| Property              | Description                                                                                          |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|
+| id                    | The conference id.                                                                                   |
+| name                  | The conference name.                                                                                 |
+| createdTime           | The time the conference was initiated, in ISO 8601 format.                                           |
+| completedTime         | The time the conference was terminated, in ISO 8601 format.                                          |
+| conferenceEventUrl    | The URL to send the Conference events.                                                               |
+| conferenceEventMethod | The HTTP method to use for the request to `conferenceEventUrl`. GET or POST. Default value is POST.  |
+| tag                   | A custom string that will be sent with this and all future callbacks.                                |
+
+**NOTE:** If the number of conferences in the account is bigger than `pageSize`, a `Link` header (format `<https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences?pageSize={pageSize}&pageToken={pageToken}&minCreatedTime={minCreatedTime}>; rel="next"`) will be returned in the response. The header contains a link which can be used to retrieve the next page of conference records.
```

Comment:
We could simplify the format example to something like "format `<url>; rel="next"` to discourage parsing the URL and just using it as is.

Reply:
Maybe also point to an official HTTP doc like [RFC-5988](https://tools.ietf.org/html/rfc5988) if that's the right one?